src/main/java/br/com/sbk/sbking/networking/kryonet/messages/GameScreenFromGameNameIdentifier.java

```diff
@@ -23,6 +23,7 @@ private GameScreenFromGameNameIdentifier() {
     gameScreenClassesOfGameNames.put("Cagando no Bequinho", CagandoNoBequinhoScreen.class);
     gameScreenClassesOfGameNames.put("King", KingScreen.class);
     gameScreenClassesOfGameNames.put("Minibridge", MinibridgeScreen.class);
+    gameScreenClassesOfGameNames.put("Mini-Minibridge", MinibridgeScreen.class);
```

Comment:
Note that both minibridges use the same screen.

Reply:
Noted